spec/system/submissions/review_spec.rb

```diff
@@ -234,6 +237,67 @@
       )
     end
 
+    scenario "coach rejects a pending submission with a feedback" do
```

Comment:
It's good to have a check on the _Undo Rejection_ operation as well, to check the DB changes